data/brands/amenity/cafe.json

```diff
@@ -3507,6 +3507,18 @@
         "takeaway": "yes"
       }
     },
+    {
+      "displayName": "Puccino's",
+      "locationSet": {"include": ["gb", "ie"]},
+      "tags": {
+        "amenity": "cafe",
+        "brand": "Puccino's",
+        "brand:wikidata": "",
+        "cuisine": "coffee_shop",
+        "name": "Puccino's",
+        "takeaway": "yes"
```

Comment:
Assumed, given its nearly all on railway platforms...

Reply:
I believe takeaway is added by default for coffee shops